airbyte-workers/src/test/java/io/airbyte/workers/process/DockerProcessFactoryTest.java

```diff
@@ -41,6 +46,29 @@
 
   private static final Path TEST_ROOT = Path.of("/tmp/airbyte_tests");
 
+  /**
+   * {@link DockerProcessFactoryTest#testImageExists()} will fail if JQ is not installed. The logs get
+   * swallowed when run from gradle. This test exists to explicitly fail with a clear error message
+   * when JQ is not installed.
+   */
+  @Test
+  public void testJqExists() throws IOException {
+    final Process process = new ProcessBuilder("jq", "--version").start();
+    final StringBuilder out = new StringBuilder();
+    final StringBuilder err = new StringBuilder();
+    LineGobbler.gobble(process.getInputStream(), out::append);
+    LineGobbler.gobble(process.getErrorStream(), err::append);
+
+    WorkerUtils.gentleClose(process, 1, TimeUnit.MINUTES);
+
+    assertEquals(0, process.exitValue(), String.format("Error while checking for JQ. STDOUT: %s STDERR: %s", out, err));
```

Comment:
Making it way more explicit?
```suggestion
    assertEquals(0, process.exitValue(), String.format("Error while checking for JQ. STDOUT: %s STDERR: %s. Please make sure jq is installed (used by testImageExists)", out, err));
```